src/server/src/main/java/io/cassandrareaper/jmx/JmxProxyImpl.java

```diff
@@ -1268,4 +1280,28 @@ public void takeColumnFamilySnapshot(
     return ((StreamManagerMBean) smProxy).getCurrentStreams();
   }
 
+  public List<Compaction> listActiveCompactions()
```

Comment:
i'm keen to stop any further "feature" additions to this class.

Can we rebase this PR onto `mck/feature-major-compaction` and this method into CompactionProxy.

Reply:
I've done this work in https://github.com/thelastpickle/cassandra-reaper/commit/e58c6116eb4cd33b7ecf05291341305e7267a38a
if you'd like to bring it in.